package.json

```diff
@@ -64,10 +64,11 @@
         "bundle": "./node_modules/.bin/webpack --config webpack.config.js",
         "analyze": "BUNDLE_ANALYSIS=\"reports/commit_$(git rev-parse --short HEAD).html\" ./node_modules/.bin/webpack --config webpack.config.js",
         "lint": "eslint .",
-        "test": "npm-run-all test-unit test-int test-e2e",
+        "test": "npm-run-all test-unit test-int test-e2e test-e2e-adhoc",
         "test-int": "cross-env NODE_PATH=src:test ava test/int",
         "test-unit": "cross-env NODE_PATH=src:test ava test/unit",
         "test-e2e": "cross-env NODE_PATH=src:test ava test/e2e/run-tests.js",
+        "test-e2e-adhoc": "cross-env NODE_PATH=src:test ava test/e2e/**/*.test.js",
```

Comment:
I we rename `run-tests.js` to run.test.js and update the `test-e2e` script to `cross-env NODE_PATH=src:test ava test/e2e/**/*.test.js`, we won't need the `adhoc` script.

Reply:
Doesn't work like that (I have tried).

Reply:
That's weird. I tried it and it worked for me. 🤔 

Let me see if I can get it working again.